test/morphometrics/test_compute_morphometrics.py

```diff
@@ -228,6 +337,20 @@ def test_draw_axon_diameter_creates_file_in_expected_location(self):
 
         assert result_path.is_file()
         result_path.unlink()
+    
+    @pytest.mark.unit
+    def test_draw_axon_diameter_creates_file_in_expected_location_with_axon_as_ellipse(self):
+        img = imageio_imread(self.test_folder_path / 'image.png')
+        path_prediction = self.test_folder_path / 'AxonDeepSeg_seg-axonmyelin.png'
```

Comment:
Should we use the standard axonmyelin_suffix here for the filename?

Reply:
yes, I've fixed it.